client-tutorial/administrative_data.md

```diff
@@ -0,0 +1,55 @@
+# The administrative data part of the tutorial
+## Henry Feldman MD, FACP, FHM, FAMIA
+## CMO for Development
+
```

Comment:
I wouldn't include your name and title in the open source doc, but your name would be in github as the contributor.